BUILD

```diff
@@ -54,9 +54,9 @@ cc_library(
         "src/google/protobuf/stubs/time.cc",
         "src/google/protobuf/wire_format_lite.cc",
     ],
+    hdrs = glob(["src/google/protobuf/**/*.h"]),
```

Comment:
Why did you rearrange these? Is there a canonical ordering? It's not properly alphabetical before or after this change.